.gitattributes

```diff
@@ -0,0 +1,2 @@
+*  					text=auto
+/tests/inputs/*	text eol=lf
```

Comment:
Why add this .gitattributes file?

Reply:
This is because most git configurations use `core.autocrlf` set to true, so when you `clone` a repo on Windows git will automatically convert all the files to use Windows line endings. This is problematic for the test input files where the test does a simple `TEST_ASSERT_EQUAL_STRING` to compare the file contents and then fails.

One alternative is to modify the test to be agnostic to line endings (likely by stripping all `\r` from the expected output). If you prefer that, I can make that update instead!